internal/ethapi/api.go

```diff
@@ -623,6 +623,24 @@ func (s *PublicBlockChainAPI) GetCode(ctx context.Context, address common.Addres
 	return code, state.Error()
 }
 
+// GetAccountInfo returns the information at the given address in the state for the given block number.
+func (s *PublicBlockChainAPI) GetAccountInfo(ctx context.Context, address common.Address, blockNr rpc.BlockNumber) (map[string]interface{}, error) {
+	state, _, err := s.b.StateAndHeaderByNumber(ctx, blockNr)
+	if state == nil || err != nil {
+		return nil, err
```

Comment:
u might wanna return state is nil error in case err is also nil

Reply:
when state is nil, return `nil, nil` is enough.